ingenialink/servo.py

```diff
@@ -893,7 +893,6 @@ def state_wait_change(self, state: SERVO_STATE, timeout: int, subnode: int = 1)
             time_diff = current_time - start_time
             if time_diff > timeout:
                 raise ILTimeoutError
-            # TODO: Remove this try-except after CAP-924 is solved.
```

Comment:
Shouldn't we have a comment here why we have the try except?
The rule is to no have "TODO". But we can keep links to future issues with possible improvements

Reply:
I don't like the idea of a link a Jira the issue here. The Jira issue has a link to the comment.